stock_partner_delivery_window/models/res_partner.py

```diff
@@ -101,6 +101,9 @@ def is_in_delivery_window(self, date):
         :return: Boolean
         """
         self.ensure_one()
+        tz = pytz.timezone(self.tz or "UTC")
```

Comment:
Question: as you are using `<res.partner>.tz` field, should we not keep `partner_tz` dependency by convenience? It adds the field on partner form view.